doc/nrf/ug_edge_impulse.rst

```diff
@@ -109,11 +115,44 @@ Complete the following steps to do this:
 
       Project ID in Edge Impulse studio dashboard
 
-#. Define the :c:macro:`EI_API_KEY_HEADER` CMake option as ``x-api-key:aaaabbbbccccdddd`` to provide the x-api-key associated with yout Edge Impulse project.
-   You can access your API keys under the :guilabel:`Keys` tab in the Edge Impulse project dashboard.
+#. Define the :c:macro:`EI_API_KEY_HEADER` CMake option (see :ref:`cmake_options`) as ``x-api-key:[ei api key]`` to provide the x-api-key associated with your Edge Impulse project.
+   To check what to provide as the *[ei api key]* value, check your API keys under the :guilabel:`Keys` tab in the Edge Impulse project dashboard.
 
    .. figure:: images/ei_api_key.png
       :scale: 50 %
       :alt: API key under the Keys tab in Edge Impulse studio
 
       API key under the Keys tab in Edge Impulse studio
+
+Downloading from a public project
++++++++++++++++++++++++++++++++++
+
+Complete the following steps to download the model from a public Edge Impulse project:
+
+1. Check the ID of the public project.
+   This ID is different from the ID shown in the Edge Impulse Studio in the :guilabel:`Project info` panel.
+   You can check the public ID by visiting the following URL:
+
+   .. parsed-literal::
+      :class: highlight
+
+      https:\ //studio.edgeimpulse.com/v1/api/*XYZ*/versions/public
+
+   Set *XYZ* to the ID shown in the Edge Impulse Studio in the :guilabel:`Project info` panel under :guilabel:`Dashboard`.
+   The ID of the public project is displayed next to the "publicProjectId" field.
+
+   .. figure:: images/ei_project_id.png
+      :scale: 50 %
+      :alt: Project ID in Edge Impulse studio dashboard
+
+      Project ID in Edge Impulse studio dashboard
```

Comment:
Duplicated the figure from line 112, but we need an updated one, so that the publicProjectId is visible.